app/scripts/extensions/nav/helpDropdown.js

```diff
@@ -17,6 +17,10 @@ angular.module('openshiftConsole')
           },
           {
             type: 'dom',
+            node: '<li class="divider"></li>'
```

Comment:
keep in mind in online we add menu items to the bottom of this menu, is that going to look weird with this divider added